commons/src/main/java/org/frankframework/util/ClassUtils.java

```diff
@@ -124,10 +124,24 @@ public static long lastModified(Class<?> aClass) throws IllegalArgumentException
 	 *
 	 * @param className A class name
 	 * @return A new instance
-	 * @exception Exception If an instantiation error occurs
+	 * @exception ReflectiveOperationException If an instantiation error occurs
+	 * @exception SecurityException If a security violation occurs
 	 */
-	public static Object newInstance(String className) throws Exception {
-		return ClassUtils.loadClass(className).getDeclaredConstructor().newInstance();
+	public static Object newInstance(String className) throws ReflectiveOperationException, SecurityException {
```

Comment:
Is deze versie van de method nog steeds nodig? Of kan deze weg?

Reply:
Deze wordt nog steeds gebruikt.